var/spack/repos/builtin/packages/benchmark/package.py

```diff
@@ -10,12 +10,13 @@ class Benchmark(CMakePackage):
     """A microbenchmark support library"""
 
     homepage = "https://github.com/google/benchmark"
-    url      = "https://github.com/google/benchmark/archive/v1.5.5.tar.gz"
+    url      = "https://github.com/google/benchmark/archive/v1.6.0.tar.gz"
```

Comment:
FYI. This change isn't technically required since Spack uses version substitution.

Reply:
Hi @tldahlgren yep true! I usually update the URL anyway (as long as it's within the same format) so that people can find the most up-to-date version. (Which I sometimes do myself when tweaking a `package.py` file locally.)